.coveragerc

```diff
@@ -1,5 +1,7 @@
 [run]
 branch = True
+# TODO: Remove the plugins_interfaces.py omission when new interfaces are used
+omit = samcli/lib/iac/plugins_interfaces.py
```

Comment:
We should add the other new files here to increase coverage.

Reply:
I added some mock tests to cover them. Everything other than this file should be fully covered.

Reply:
I added some comments to make this more clear.